src/analytics/config.ts

```diff
@@ -1,66 +1,77 @@
-import { NativeInstanceConfig } from '../nativeInstance';
-
 /**
- * Available cdn provider options for AnalyticsConfig.
+ * Object used to configure the build-in analytics collector.
  */
-export enum CdnProvider {
-  BITMOVIN = 'bitmovin',
-  AKAMAI = 'akamai',
-  FASTLY = 'fastly',
-  MAXCDN = 'maxcdn',
-  CLOUDFRONT = 'cloudfront',
-  CHINACACHE = 'chinacache',
-  BITGRAVITY = 'bitgravity',
+export interface AnalyticsConfig {
```

Comment:
is it intended to not support `backendUrl` and `retryPolicy` of the `AnalyticsConfig`? :thinking:

Reply:
Yes, those are features that where not supported before.
I mainly wanted to migrate to the new API, without adding new stuff. I don't see a particular need for them at the moment and adding them should be straight forward then.